explainaboard/info.py

```diff
@@ -55,17 +55,57 @@ def from_dict(cls, data_dict: dict) -> Performance:
         return cls(**{k: v for k, v in data_dict.items() if k in field_names})
 
 
+@dataclass
+class BucketCase:
+    sample_id: str
+
+    @classmethod
```

Comment:
Cool.
* I named it as `dict_conv` to keep consistent with other lines of functions. For example: https://github.com/neulab/ExplainaBoard/blob/aaf91a57f4a5c143a6f6c2aec33a81a3e51690b8/explainaboard/info.py#L88
* the reason why I keep it (even it just returns itself) is to pave the way for future expansion (for example, if the class include some members with dataclass type.

It's still good to consider introducing a unified & general interface for `from_dict`
I think you have implemented on: https://github.com/neulab/ExplainaBoard/blob/aaf91a57f4a5c143a6f6c2aec33a81a3e51690b8/explainaboard/utils/serialization.py#L4
(it seems that it could n't deal with the nested dataclass cases)  Set up an issue for future refinement.

Reply:
https://github.com/neulab/ExplainaBoard/issues/302